openacc/nemolite2d.f90

```diff
@@ -759,31 +759,49 @@ SUBROUTINE next
 
           call timer_start(idxt, label='Next')
 
-! kernel  un updating
+! kernel  un updating   
+          !$acc parallel            
```

Comment:
Suggested change: use a single parallel region with multiple loops

Reply:
Have you looked into the data transfer costs for this? You could copy the `present(...)` and `default(none)` clauses from the momentum loop and implement the `private(...)` for the loop variables.

You'd need a `copyin(ua, va, ssha, etc.)` and `copyout(un, vn, sshn, etc.)` for all the variables that are _not constant_ and are read from/written to/both.